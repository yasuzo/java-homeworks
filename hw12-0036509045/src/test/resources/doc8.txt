{$ for var "\"this\t\t \\ is\r string\"\n" var2$}
{$end$}

{$for var1 "Joe\\ \"Long\" Smith" var2  var3 $}
{$end$}