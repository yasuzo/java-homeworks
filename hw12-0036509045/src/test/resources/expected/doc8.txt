{$ FOR var "\"this\t\t \\ is\r string\"\n" var2 $}
{$ END $}

{$ FOR var1 "Joe\\ \"Long\" Smith" var2 var3 $}
{$ END $}